book.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'item'
+
+class Book < Item
+  attr_accessor :publish, :cover_state
+
+  def initialize(archieved, publish, cover_state, publish_date)
```

Comment:
Kindly, check the typo: archieved, and it seems like the requirements said 'publisher' and not 'publish'.